HomeTask6/app/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="footer">
+    <h5 style="float: left;margin-left: 20px;">Footer</h5>
+    <h5>Powered By NewsAPI&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h5>
```

Comment:
use template literals instead of declaring so many &nbsp;

Reply:
Done